pkg/controller/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -541,10 +541,12 @@ func TestSyncMurStatusWithUserAccountStatuses(t *testing.T) {
 		// Get the notification resource and verify it
 		notification := toolchainv1alpha1.Notification{}
 		hostClient.Get(context.TODO(), namespacedName(mur.Namespace, userAccount.Name+"-provisioned"), &notification)
-		assert.Equal(t, notification.Spec.UserID, userAccount.Spec.UserID)
-		assert.Equal(t, notification.Spec.Template, "userprovisioned")
-		assert.Equal(t, notification.Name, userAccount.Name+"-provisioned")
-		assert.Equal(t, notification.Namespace, mur.Namespace)
+		assert.Equal(t, userAccount.Spec.UserID, notification.Spec.UserID)
+		assert.Equal(t, "userprovisioned", notification.Spec.Template)
+		assert.Equal(t, userAccount.Name+"-provisioned", notification.Name)
+		assert.Equal(t, mur.Namespace, notification.Namespace)
+		assert.Equal(t, "MasterUserRecord", notification.OwnerReferences[0].Kind)
```

Comment:
Worth checking that `len(notification.OwnerReferences) == 1` to avoid panics.

Reply:
good point! fixed here: bc86b0a715e88f48ba7e7ee2783d6684df92ac00